lib/kitchen/directions/bake_chapter_introductions/bake_chapter_objectives.rb

```diff
@@ -32,12 +32,16 @@ def bake_as_note(chapter:)
     end
 
     def add_chapter_objectives(chapter:, options: {})
-      options.reverse_merge!(numbering_options: { mode: :chapter_page, separator: '.' })
+      options.reverse_merge!(
+        numbering_options: { mode: :chapter_page, separator: '.' },
+        cases: false)
       chapter.non_introduction_pages.map do |page|
         number = page.os_number(options[:numbering_options])
+        title_text = page.title_children.map(&:text).join(' ')
+        label = I18n.t(:section_link_desc, link_text: "#{number} #{title_text}")
```

Comment:
I see `section_link_desc` added to Spanish and Polish locales, but not the English locale, is that a problem?